tests/test_CIR.py

```diff
@@ -0,0 +1,73 @@
+import unittest
```

Comment:
I'm not familiar with the unittest package, but the tests look good. Feel free to also check out [pytest](https://docs.pytest.org/en/6.2.x/) sometime.

Reply:
`pytest` can actually work with `unittest` tests, I'm using it in [`python-package` workflow](https://github.com/xtrojak/pyMSPannotator/blob/main/.github/workflows/python-package.yml).